docs/DataWorkflows/FetchingYourData.md

```diff
@@ -246,6 +243,212 @@ Critically, the reference to the EXACT version of the file you used in your
 analysis will be fetched if you need to
 [access the data again](#clone-the-static-data).
 
+# Unzipping your data
+
+Data on PMACS might be zipped, or unzipped. In the case of unzipped data, you should be good to go following the guidelines on finding files you need [here](./FetchingYourData.md)! If your data is zipped, you might need a little more work.
+
+The following walkthrough should help you unzip your data. These lines of code can be customized per your needs to unzip some or all subjects and then some or all files: 
+
+Before running a loop for all participants of a dataset, you probably first want to figure out what data is available, and which files you’d want to get. Most datasets are zipped, so here is a walkthrough for a single subject. In this case, we’re interested in getting fMRIPrep data from PNC, which is zipped and stored on PMACS. 
+
+Once data is cloned from PMACS, let’s `cd` into to it to see what’s there: 
+
+```bash
+sub-DobbySockster_fmriprep-20.2.3.zip@     
+sub-HarryPotter_fmriprep-20.2.3.zip@
+sub-NevilleLongbottom_freesurfer-20.2.3.zip@   
+```
+
+The folder contains a bunch of folders and simlinks to zipped participants. 
+
+### Step 2: Get and unzip a participant
+
+The next step is to get and unzip data for one participant to figure out what files are available. 
+
+```bash
+# make directory to store unzipped files
+mkdir -p /cbica/projects/hogwarts/data/PNC/PNC_fMRIPrep_v20_2_3/zipped
+mkdir -p /cbica/projects/hogwarts/data/PNC/PNC_fMRIPrep_v20_2_3/unzipped
+
+# define a specific participant file 
+file="sub-NevilleLongbottom_fmriprep-20.2.3.zip"
+
+# get the file for one specific participant
+datalad get $file -J 8 # runs the job on 8 threads
+```
+
+The output of datalat get will look like this: 
+
+```bash
+It is highly recommended to configure Git before using DataLad. Set both 'user.name' and 'user.email' configuration variables.
+Total:   0%|                                                                                | 0.00/1.19G [00:00<?, ? Bytes/s]
+dumbledore@bblsub.pmacs.upenn.edu's password:
```

Comment:
if we want to follow the structure for the unzipped dataset, here bblsub should be replaced with [login node name]
(and wherever else is relevant below)